content/03.use-cases-a.md

```diff
@@ -1,9 +1,11 @@
 ## GitHub in EcoEvo examples (Part 1)
 
 ### Storing and archiving version-controlled data
-*contributors to this section:* 
-Another potential use case/user perspective: Some people are just using GitHub to  backup their data, use their code on differnt machines. Just push and pull (Box 1) from their own repo. 
-GitHub integrates with Zenodo, a popular, free data archiving service funded by CERN. After linking your GitHub account to Zenodo and turning on archiving, any time a release (Box 1) is made, a snapshot of the entire repository is archived in Zenodo with a versioned, citable DOI
+*contributors to this section: Dylan Gomes*
+
+Many researchers often start their use of GitHub to backup their working data and code to a remote server (Just push and pull, see Box 1, from their own repo). This saves the user time from backing up data and code on their own portable devices, such as hard drives. This also offers some peace of mind, as this information is retrievable even if one's laptop ends up at the bottom of a lake. Thus, an additional benefit of this 'cloud' storage is that one's GitHub repository can be accessed by any machine with internet access, allowing the user to be more mobile if they wish to both work from home and the office from different computers. Each time a user pushes changes to their repository, GitHub tracks what these changes are and stores this history. This feature allows for version control, such that users can re-visit previous versions of data and code. This is particularly useful if a mistake has been made where a user has unknowingly overwritten or deleted information that would otherwise be irretrievable without GitHub having saved that information.
+
+GitHub also integrates with Zenodo, a free, long-term data archiving service funded by CERN. After linking your GitHub account to Zenodo and turning on archiving, any time a release (Box 1) is made, a snapshot of the entire repository is archived in Zenodo with a versioned, citable DOI (see `### Making code citable` below for more information).
```

Comment:
I think [@doi:10.2777/298228](https://data.europa.eu/doi/10.2777/298228) might be an appropriate citation for the first sentence here?

Reply:
I like this suggestion for the sentence on line 14!